docs/tutorials/forum-application/3.send-message.md

```diff
@@ -0,0 +1,381 @@
+# Send Message
+
```

Comment:
Please, include a paragraph somewhere saying that, for this code to be complete, we need to add signatures to transactions (both messages and ban TXs).
If would also be good IMO to add comments in the code pointing where the signature verification would take place (although I'm not sure that would help understanding).

Reply:
Is this mentioned somewhere in the updated PR?